test/system/liquidation.test.js

```diff
@@ -0,0 +1,127 @@
+  const auctionLength = 86400 // 24h
+  // deposit lot size is 5 BTC
+  const lotSize = to1e18(5)
+  // signers have bonded 290.81 ETH
+  const bondedAmount = BigNumber.from("290810391624000000000")
+  // 66% of the deposit is exposed on auction in the liquidation moment
+  const bondedAmountPercentage = BigNumber.from("66")
+
+  let tbtcToken
+  let signerBondsSwapStrategy
+  let coveragePool
+  let riskManagerV1
+  let tbtcDeposit
+
+  let bidder
+
+  before(async () => {
+    await resetFork(startingBlock)
+
+    tbtcToken = await ethers.getContractAt("IERC20", tbtcTokenAddress)
+
+    const SignerBondsSwapStrategy = await ethers.getContractFactory(
+      "SignerBondsEscrow"
+    )
+    signerBondsSwapStrategy = await SignerBondsSwapStrategy.deploy()
+    await signerBondsSwapStrategy.deployed()
+
+    const Auction = await ethers.getContractFactory("Auction")
+
+    const masterAuction = await Auction.deploy()
+    await masterAuction.deployed()
+
+    const CoveragePoolStub = await ethers.getContractFactory("CoveragePoolStub")
+    coveragePool = await CoveragePoolStub.deploy()
+    await coveragePool.deployed()
+
+    const RiskManagerV1 = await ethers.getContractFactory("RiskManagerV1")
+    riskManagerV1 = await RiskManagerV1.deploy(
+      tbtcToken.address,
+      coveragePool.address,
+      signerBondsSwapStrategy.address,
+      masterAuction.address,
+      auctionLength
+    )
+    await riskManagerV1.deployed()
+
+    tbtcDeposit = await ethers.getContractAt("IDepositStub", depositAddress)
+
+    bidder = await impersonateAccount(bidderAddress)
+  })
+
+  describe("when auction has been fully filled", () => {
+    let auction
+    let tx
+    let bidderInitialBalance
+
+    before(async () => {
+      await tbtcDeposit.notifyRedemptionSignatureTimedOut()
+      await riskManagerV1.notifyLiquidation(tbtcDeposit.address)
+
+      const auctionAddress = await riskManagerV1.depositToAuction(
+        tbtcDeposit.address
+      )
+      auction = new ethers.Contract(auctionAddress, Auction.abi, bidder)
+      await tbtcToken.connect(bidder).approve(auction.address, lotSize)
+      bidderInitialBalance = await tbtcToken.balanceOf(bidder.address)
+      tx = await auction.takeOffer(lotSize)
+    })
+
+    it("should close auction", async () => {
+      expect(await auction.isOpen()).to.be.false
+    })
+
+    it("should liquidate the deposit", async () => {
+      // Auction bidder has spend their TBTC.
+      const bidderCurrentBalance = await tbtcToken.balanceOf(bidder.address)
+      expect(bidderInitialBalance.sub(bidderCurrentBalance)).to.equal(lotSize)
+
+      // Deposit has been liquidated.
+      expect(await tbtcDeposit.currentState()).to.equal(11) // LIQUIDATED
+    })
+
+    it("should swap signer bonds", async () => {
+      // No funds should last on the risk manager contract.
+      await expect(tx).to.changeEtherBalance(riskManagerV1, 0)
+
+      // All funds should be moved to the signer bonds processor contract.
```

Comment:
s/signer bonds processor contract/signer bonds swap strategy

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/53/commits/75376a9a2fb20fbe099174b2bb224fa34c48f7c7